services/secret-service/server/src/main/resources/secrets.json

```diff
@@ -36,6 +36,6 @@
   {
     "type": "GDRIVE",
     "secretId": "gdrive-cred",
-    "credentialsJson": ""
+    "credentialsJson": "{\n  \"type\": \"service_account\",\n  \"project_id\": \"annular-reef-275521\",\n  \"private_key_id\": \"c3794257f4bb324c77e172142b84ecc1fa2a258a\",\n  \"private_key\": \"-----BEGIN PRIVATE KEY-----\\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQCZujULI/vWaJi9\\nJD7Zi9I1h9l6FMCuKyAcYEZbyQR7kBR3xavTbZpT1wD7c0xAffq/CraZrBJ3eh0j\\nLtlywhJesabGA7XtoG5qt190voWoofrYaD/ngvxc0VCzZdAU8AbIZACXlyPYmNsU\\n/U7lFebk4XSeLhQK2f3MaQMbazxyY8iNrI4fxcAZeP8NbpLGPb9vRFXOePtmwhlu\\nJa2heDSKJIdveP1NAEXWx4Xq0yJZSV39sCWZThGTXuWBw8qjRqSIzAV+niQspqIM\\n07HDt82+EKeGq8CqJ3bEbQaTBtydcDLJ6C+M9JJzNBo1uWpMF0D96BMqvYfEOlgp\\n4NivYMdfAgMBAAECggEADLgl9GaFS0+7N4cctiZF/czgNyQpbx+9V+u1VpnSx6B7\\nuQ8/cM50QvSarbNz/zANQ6pQwnVEz+LoL0WMk1ywZWpgiKi88EziZe0HNmMfLv08\\nApTZmL+zPsDI4xcM+PaFMSZpECHewuie0+VDllvSD2UCH6ZjXSR9ReD4c1fTmAk9\\np4jXC/sAgmRymhR5y73TULM9Wl98O3ercZ8cdglkxKXBO51BicwJmnjY8xI390e/\\nkjfFJonhaz6uI3uNdjLV4aIiZZ6I8v07JLM6HaMVdegvqQDzx+fwoVntoMYeMxX5\\nV9QoIt1z134H2IvzPfP0LiTeyLRMt1ALzgi9/3z9wQKBgQDSgNsRIjVJeX+Qfbgn\\nnqJWGwmBlxPTzvlCxUI7n7uh+mhsjxIerRFopvLs8cvqdEJ5dmEcHnKwmqOcJZOk\\nCdR+KcoMDdR0VQi6UWZmRHkEp//lZ6hs1R5/1GQqHbNFLQ73DxFwu6l3ZcpBRvxW\\n72DAO4EK5Ea8M7qlMTkpJLtqnwKBgQC68/BECgyauPNprHvcDICd35HP6jaZldp/\\n32oKBKuoHHgCW1bJhOC0m9nRasA2MfJW5ech66RPPPJE2ANBi3sp0ah8LbpvQcL1\\nl3travuBccPrcfT2RGsRFB/aj3IvcqNCxgcdSP3uFjlqty5kKveZqI1e7U19VL3G\\nj1lutd4rQQKBgQChHbEOlyE0AIOLoz39TSSxRPZssqahWmxcF8EPA75IRvIADS1w\\nP0Tw1b9fbd6Gr79lWh8Pt5pO471QOU6cok161x9nRHqVI7SPbtgydDwwPSnWexVl\\nX2V9r6GOvrbEvq0SmfzeTTRzR3ybo+pl28gwuhqVrI0bZXh5EYjPnea4vwKBgB/s\\nfc+wg5x8Dpyx5miZ/fB3d9ewH+i0DR6e0OWrDIDlc5y7aiN5PEG7B0PaXb4jY0Vx\\ndrl1GpapSpl05YFwxvRKJ+IkkNxkt1zmpB2gpBfjjFgNGo5e+oVtEU9qBU6S3cwO\\nMrLOfwYD54QQiaxzJdqoTJO0I/JN4SYh3lyoU6rBAoGAdBoFzvPeFHjjEsJ4H207\\nPm0DTpDaUXPxVKw+BBAlqkORpMOA0NQO/LVrFVWH6UJKGdCx/NB/Odu4XEq4lpP+\\nezk+1jd9oLj+JynpC120VQxxlJPcZaCxjKz+YdXG9cNmMFv7DAM3B/KKxScv0BJc\\n7Udbct6Elxc01T+9Atl1M5A=\\n-----END PRIVATE KEY-----\\n\",\n  \"client_email\": \"nsaproject@annular-reef-275521.iam.gserviceaccount.com\",\n  \"client_id\": \"114461563175161685388\",\n  \"auth_uri\": \"https://accounts.google.com/o/oauth2/auth\",\n  \"token_uri\": \"https://oauth2.googleapis.com/token\",\n  \"auth_provider_x509_cert_url\": \"https://www.googleapis.com/oauth2/v1/certs\",\n  \"client_x509_cert_url\": \"https://www.googleapis.com/robot/v1/metadata/x509/nsaproject%40annular-reef-275521.iam.gserviceaccount.com\"\n}"
```

Comment:
@nbafna04 your credentialsJson object got pushed to this commit. Please revoke it.

Reply:
Thanks for pointing it out. We have removed those credentials and deleted that user.